pkg/endpoints/pipeline.go

```diff
@@ -11,23 +13,22 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package endpoints
```

Comment:
This is a nit, but please leave `package endpoints` below the copyright. It looks like we're kind of inconsistent with this in our repo, but I think most of the `package endpoints` lines are below the copyright. Also, I checked a file in the kubernetes and tektoncd/pipeline repos and they both have the `package NAME` below the copyright: https://github.com/tektoncd/pipeline/blob/master/pkg/apis/pipeline/v1alpha1/artifact_bucket.go and https://github.com/tektoncd/pipeline/blob/master/pkg/apis/pipeline/v1alpha1/artifact_bucket.go.

Reply:
Yeah initially we had moved them above the copyright as the go linter was complaining that the package should be at the top of the file but it looks like the convention in projects we're working around is to keep the copyright first